test/patterns/bidirect_index/BidirectIndexIncreaseDecreaseAssignment.java

```diff
@@ -0,0 +1,7 @@
+class BidirectIndexIncreaseDecreaseAssignment {
+    void bidirectIndexIncreaseDecreaseAssignment() {
+        int i = 0;
+        i++;
+        i = i - 1;
```

Comment:
add tests with hiding the scope  (for cycle, while cycle, any other scope)
```
i = 0;
while(true) {
  ++i;  // fake case
}
--i; // fake case
```

Reply:
@lyriccoder Added the tests. Note that the case with `while` is a pattern, because the while does not hide the variable